wondrous-app/components/Common/Image/index.tsx

```diff
@@ -29,8 +47,9 @@ export const SafeImage = (safeImageArgs: SafeImageArgs) => {
     objectPosition = 'center',
     ...props
   } = safeImageArgs;
+
   const [imgUrl, setImageUrl] = useState(null);
-  const hasProtocol = typeof src === 'string' && (src?.startsWith('https') || src?.startsWith('file://'));
+  const hasProtocol = typeof src === 'string' && (src?.startsWith('http') || src?.startsWith('file://'));
```

Comment:
Think we would only want to accept https

Reply:
We always use HTTPS except for localhost, I've changed to the http, so the storybook could load images locally.